circle.yml

```diff
@@ -1,36 +1,36 @@
-machine:
-  java:
-    version: oraclejdk8
-  environment:
-    TERM: dumb
-  post:
-    # install JCE Unlimited Strength Jurisdiction Policy Files
-    - |
-      wget -qO- --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" "http://download.oracle.com/otn-pub/java/jce/8/jce_policy-8.zip" | bsdtar -xvf- -C /tmp && sudo cp /tmp/UnlimitedJCEPolicyJDK8/* $JAVA_HOME/jre/lib/security/
-
-general:
-  artifacts:
-    - encrypted-config-value/build/libs
-    - encrypted-config-value-bundle/build/libs
-    - encrypted-config-value-bundle-dropwizard1/build/libs
-
-dependencies:
-  override:
-    - ./gradlew resolveConfigurations
-
-test:
-  override:
-    - ./gradlew check
-
-deployment:
-  # disable until we work through OJO publishing issues with Bintray
-  #develop:
-  #  branch: develop
-  #  owner: palantir/encrypted-config-value
-  #  commands:
-  #   - ./gradlew -i artifactoryPublish
-  release:
-    tag: /[0-9]+(\.[0-9]+)+/
-    owner: palantir
-    commands:
-      - ./gradlew -i bintrayUpload
+version: 2
+jobs:
+  build:
+    working_directory: /root/encrypted-config-value
+    environment:
+      ARTIFACTS_DIR: /tmp/artifacts
+      TESTS_DIR: /tmp/tests
+    docker:
+      - image: nmiyake/java:alpine-oracle-jdk-8-t3
```

Comment:
Yup totally cool with that, but even with our automation creating a Palantir GitHub repo isn't instant, so just a stopgap for now. Also not aware of any existing public Palantir repos that publish Docker images (otherwise, could just piggy-back on that).

Are you OK with me requesting a `java-docker-images` GitHub.com repo? Or I guess we could also publish from `palantir.gh.build` into public Docker Hub